distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -221,6 +222,53 @@ Complete the following procedure now that several queries have been executed.
 Based on the browser's configuration, the .xls file will be downloaded or automatically displayed in Excel.
 ====
 
+
+
+
+=== Configuration Export/Import
+==== Saving Configurations
```

Comment:
There is a general configuring section below (line 750-ish) that details configuring with and without admin UI.